client/package.json

```diff
@@ -16,6 +16,7 @@
     "web-vitals": "^0.2.4"
   },
   "scripts": {
+    "dev": "chmod +x ./env.sh && ./env.sh && cp env-config.js ./public/ && react-scripts start",
```

Comment:
I usually use a verb for the npm scripts. Is this about starting the app in development mode? If yes, then we can have `start:dev`

Reply:
I just changed it to `start`, actually. Does that work?